packages/ui-components/src/components/tree-item/readme.md

```diff
@@ -174,3 +174,5 @@ graph TD;
 ```
 
 ----------------------------------------------
+
```

Comment:
?

Reply:
this is automatically generated at build